media-gfx/drawpile/drawpile-2.0.10.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A collaborative drawing program"
+DESCRIPTION="Drawpile is a drawing program that lets you share \
+the canvas with other users in real time."
+HOMEPAGE="https://drawpile.net/"
+COPYRIGHT="2006-2018 Calle Laakkonen"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/drawpile/Drawpile/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="0ff79e9f95ca02ff016eed3118a83ca62e80a1ca93fe11e3fb8ecd22c1fe37a9"
+SOURCE_FILENAME="Drawpile-$portVersion.tar.gz"
+SOURCE_DIR="Drawpile-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	drawpile$secondaryArchSuffix = $portVersion
+	app:Drawpile = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
```

Comment:
Thanks @hako for the update.
I was going to approve and merge the PR but then realized I had forgotten to check the `REQUIRES`.
It looks like Drawpile does not directly require 2 libs: `libGL` and `libz`:
```bash
~> objdump -x /system/apps/Drawpile/* | grep NEEDED | sort -iu | grep -Evw 'libroot|libnetwork'
  NEEDED               libgcc_s.so.1
  NEEDED               libgif.so.7
  NEEDED               libKF5Archive.so.5
  NEEDED               libminiupnpc.so.17
  NEEDED               libQt5Core.so.5
  NEEDED               libQt5Gui.so.5
  NEEDED               libQt5Multimedia.so.5
  NEEDED               libQt5Network.so.5
  NEEDED               libQt5Sql.so.5
  NEEDED               libQt5Svg.so.5
  NEEDED               libQt5Widgets.so.5
  NEEDED               libsodium.so.23
  NEEDED               libstdc++.so.6
```

Although the names are not case-sensitive I think it might be nice to make the caps match, and to explicitly add a few `lib:libQt5*` which are not yet mentioned:
```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libgif$secondaryArchSuffix
-	lib:libGL$secondaryArchSuffix
-	lib:libkf5archive$secondaryArchSuffix
+	lib:libKF5Archive$secondaryArchSuffix
 	lib:libminiupnpc$secondaryArchSuffix
 	lib:libQt5Core$secondaryArchSuffix
 	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Multimedia$secondaryArchSuffix
 	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Sql$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
 	lib:libsodium$secondaryArchSuffix
-	lib:libz$secondaryArchSuffix
 	"
```
I'm checking this right now, with `haikuporter -S drawpile`.

BTW, don't worry about `libgcc_s.so.1` and `libstdc++.so.6`. haikuporter is not complaining about them, so my guess is that we don't need to mention `lib:libgcc_s$secondaryArchSuffix` and `lib:libstdc++$secondaryArchSuffix`.